meson.build

```diff
@@ -4004,6 +4004,20 @@ public_programs += executable(
         install_rpath : rootpkglibdir,
         install : true)
 
+if conf.get('HAVE_QRENCODE') == 1
+        public_programs += executable(
+                'systemd-bsod',
+                'src/journal/bsod.c',
+                include_directories : includes,
+                link_with : [libshared],
+                dependencies : [userspace,
+                                versiondep,
+                                libqrencode],
+                install_rpath : rootpkglibdir,
+                install_dir : rootlibexecdir,
```

Comment:
You'll need to rebase on top of the latest main and change this part to:

```suggestion
                install_rpath : pkglibdir,
                install_dir : libexecdir,
```

to make the CIs happy again.

Reply:
> You'll need to rebase on top of the latest main and change this part to:
> 
> to make the CIs happy again.

Awesome 🤣🤣. Thank you sir!